<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>logo.png</normaloff>logo.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>4.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="statusTip">
       <string/>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="File">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="separator"/>
    <addaction name="newFile"/>
    <addaction name="separator"/>
    <addaction name="openFile"/>
    <addaction name="separator"/>
    <addaction name="encoding"/>
    <addaction name="separator"/>
    <addaction name="save"/>
    <addaction name="separator"/>
    <addaction name="saveAs"/>
    <addaction name="separator"/>
    <addaction name="exit"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="Edit">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <widget class="QMenu" name="menu_F">
     <property name="title">
      <string>查找和替换(&amp;F)</string>
     </property>
     <addaction name="quickFind"/>
     <addaction name="separator"/>
     <addaction name="quickReplace"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="menu_F"/>
    <addaction name="separator"/>
    <addaction name="undo"/>
    <addaction name="separator"/>
    <addaction name="redo"/>
    <addaction name="separator"/>
    <addaction name="cut"/>
    <addaction name="separator"/>
    <addaction name="copy"/>
    <addaction name="separator"/>
    <addaction name="paste"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="Run">
    <property name="title">
     <string>运行</string>
    </property>
    <addaction name="separator"/>
    <addaction name="compile"/>
    <addaction name="separator"/>
    <addaction name="run"/>
    <addaction name="separator"/>
    <addaction name="CandR"/>
    <addaction name="separator"/>
    <addaction name="syntaxCheck"/>
    <addaction name="separator"/>
    <addaction name="stop"/>
   </widget>
   <addaction name="File"/>
   <addaction name="Edit"/>
   <addaction name="Run"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="newFile"/>
   <addaction name="openFile"/>
   <addaction name="save"/>
   <addaction name="undo"/>
   <addaction name="redo"/>
  </widget>
  <action name="newFile">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/images/new.jpg</normaloff>:/image/images/new.jpg</iconset>
   </property>
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="openFile">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/images/open.jpg</normaloff>:/image/images/open.jpg</iconset>
   </property>
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="saveAs">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/images/saveAs.jpg</normaloff>:/image/images/saveAs.jpg</iconset>
   </property>
   <property name="text">
    <string>另存为(&amp;A)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="exit">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/images/exit.jpg</normaloff>:/image/images/exit.jpg</iconset>
   </property>
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="undo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/images/undo.jpg</normaloff>:/image/images/undo.jpg</iconset>
   </property>
   <property name="text">
    <string>撤销(&amp;U)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="copy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/images/copy.jpg</normaloff>:/image/images/copy.jpg</iconset>
   </property>
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="cut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/images/cut.jpg</normaloff>:/image/images/cut.jpg</iconset>
   </property>
   <property name="text">
    <string>剪切(&amp;T)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="compile">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/images/run.jpg</normaloff>:/image/images/run.jpg</iconset>
   </property>
   <property name="text">
    <string>编译</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F5</string>
   </property>
  </action>
  <action name="paste">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/images/paste.jpg</normaloff>:/image/images/paste.jpg</iconset>
   </property>
   <property name="text">
    <string>粘贴(&amp;P)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="encoding">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/images/config.jpg</normaloff>:/image/images/config.jpg</iconset>
   </property>
   <property name="text">
    <string>编码设置</string>
   </property>
  </action>
  <action name="save">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/images/save.jpg</normaloff>:/image/images/save.jpg</iconset>
   </property>
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="redo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/images/redo.jpg</normaloff>:/image/images/redo.jpg</iconset>
   </property>
   <property name="text">
    <string>重做(&amp;R)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="quickFind">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/images/find.jpg</normaloff>:/image/images/find.jpg</iconset>
   </property>
   <property name="text">
    <string>快速查找(&amp;F)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="quickReplace">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/images/replace.jpg</normaloff>:/image/images/replace.jpg</iconset>
   </property>
   <property name="text">
    <string>快速替换(&amp;R)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="run">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
  </action>
  <action name="CandR">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>编译运行</string>
   </property>
  </action>
  <action name="syntaxCheck">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>语法检查</string>
   </property>
  </action>
  <action name="stop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>停止执行</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
